src/css/style.css

```diff
@@ -1,26 +1,52 @@
+:root {
+  --main-font: 'Franklin Gothic Medium', 'Arial Narrow', arial, sans-serif;

```

Comment:
### I will use this section of your CSS file to highlight the OPTIONAL changes.
- [ ] [OPTIONAL] When I run the app I'm not able to see any scores, kindly make it possible for the user to see all the existing scores when you enter the page. 😄